media/extra_grf/shore.nfo

```diff
@@ -1,6 +1,4 @@
 //
```

Comment:
This line can probably be removed. Seems common to all the nfo files

Reply:
Done